src/apps/relay/mainrelay.c

```diff
@@ -3148,27 +3142,6 @@ static void set_ctx(SSL_CTX** out, const char *protocol, const SSL_METHOD* metho
 	{
 		int op = 0;
 
-#if defined(SSL_OP_NO_SSLv2)
```

Comment:
Support for TLS versions here was duplicated
Only set in one place (so removing here)